### PR TITLE
ref: Always use event.title instead of getting title from metadata

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Comment by lynnagara** [comment]
@mitsuhiko As far as I know we'll always have a title for an event now, and we no longer need to look inside metadata to try and construct a title. Is that accurate?

File: src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx [hunk 1]
```diff
@@ -76,7 +58,7 @@ class EventsTableRow extends React.Component {
             <GlobalSelectionLink to={link}>
               <DateTime date={event.dateCreated} />
             </GlobalSelectionLink>
-            <small>{(this.getEventTitle(event) || '').substr(0, 100)}</small>
+            <small>{event.title.substr(0, 100)}</small>
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx]: This matches how `app/components/eventOrGroupTitle` is implemented now too.

**Comment by mitsuhiko** [comment]
@lynnagara yep, that is correct. All postgres based events always have title since those are in case of doubt calculated on the fly. Snuba does not calculate dynamically but uses the persisted one which should be fine at this point since we're I believe well +90 days past this change.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx]: Good catch, will tackle that one too.

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx]: Actually that one might have to stay for now, we need a subtitle as well there.

**Pull request merged**